codecov.yaml

```diff
@@ -27,6 +27,14 @@ coverage:
         target: auto
         flags:
           - e2e-tests
+      antrea-kind-e2e-tests:
+        target: auto
+        flags:
+          - kind-e2e-tests
```

Comment:
But I can see coverage data of kind-e2e-tests in other PRs, what does the configuration affect?

Reply:
before I made the change, I didn't see any data for kind-e2e-tests, I believe it has been overwritten in Codecov by this PR. https://app.codecov.io/gh/antrea-io/antrea/settings/yaml, we need this flag so Codecov can analysis and group the data in right flag.

Reply:
for the concurrency change, due to all integration jobs are running in one integration VM, it may have conflicts if both auto integration PR and manual integration PR being trigger at the same time. not sure if @lzhecheng  have encountered such failure?